public/src/css/common.css

```diff
@@ -719,6 +719,16 @@ a:hover {
   z-index: 9999;
 }
 
+.pkdisplay { display: none;}
+   @media only screen
+   and (min-device-width : 320px)
+   and (max-device-width : 900px){ .pkdisplay { display: inline;}}
```

Comment:
Same as above + it's not clean when all in 1 line, with proper formatting is more readable:
```
@media only screen
and (min-width: 320px)
and (max-width: 900px) {
  .pkdisplay {
    [...]
  }
}
```